kinesis/kinesis.go

```diff
@@ -194,13 +194,37 @@ func newPutRecordsClient(roleARN string, awsRegion string, kinesisEndpoint strin
 		return nil, err
 	}
 
-	svcConfig := &aws.Config{}
+	var svcSess = sess
+	var svcConfig *aws.Config
```

Comment:
https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/pull/102/files#r497985487

Reply:
Thanks. Will modify it.